test/test_sourcemap/expected.css

```diff
@@ -6,3 +6,4 @@
 .bla {
   color: green;
 }
+
```

Comment:
This was wrong before – now it is correct. Compare to [the according faucet-pipeline-js test](https://github.com/faucet-pipeline/faucet-pipeline-js/blob/master/test/cli/test_sourcemap/expected.js#L16)